### PR TITLE
[IMPROVEMENT] Add documentation +  Update assets

### PR DESCRIPTION
**Pull request by TheFaR8**
Please prefix your pull request with one of the following: **[FEATURE]** **[FIX]** **[IMPROVEMENT]**.

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [contributors guide](https://github.com/CCExtractor/sample-platform/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

**My familiarity with the project is as follows (check one):**

- [ ] I have never used the project.
- [ ] I have used the project briefly.
- [x] I have used the project extensively, but have not contributed previously.
- [ ] I am an active contributor to the project.

---

Added more documentation (but still keeping the old one) to fulfill #21 

And also updated the assets of sample-platform (JQuery, Foundations, what-input) to the latest version. (Fulfill #108)

Any feedback would be appreciated 
Thanks :)

### CODE REVIEW

**Comment by saurabhshri** [comment]
@TheFaR8 My last comment about documentation still remains same. Regarding assets, do you have a local copy of Sample Platform built? Just want to verify that updating these assets don't break anything.

File: mod_deploy/controllers.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def decorated_function(*args, **kwargs):
         return decorated_function
     return decorator
 
-
+# Check github deploy signature key (valid or not)
```
Comment (canihavesomecoffee) [inline mod_deploy/controllers.py]: This is not the correct way to document methods. You need to use docstrings (with """) inside the def's instead.

File: mod_sample/media_info_parser.py [hunk 2]
```diff
@@ -27,6 +27,7 @@ def __init__(self, sample):
                 try:
                     doc = xmltodict.parse(fd.read())
                 except Exception:
+                    # Show error if Mediainfo can't be open
```
Comment (canihavesomecoffee) [inline mod_sample/media_info_parser.py]: These comment's don't help a lot.

File: mod_upload/controllers.py [hunk 3]
```diff
@@ -308,6 +308,7 @@ def process_id(upload_id):
                                 '{notes}\n*Description:*\n' \
                                 '{desc}'.format(notes=form.notes.data,
                                                 desc=form.IssueBody.data)
+                        # Make the issue on GitHub issue tracker
```
Comment (canihavesomecoffee) [inline mod_upload/controllers.py]: The line below ("make_github_issue") is already descriptive enough. I don't see the need for an additional comment here.

**Pull request closed without merging**